packages/mc-scripts/src/commands/login.js

```diff
@@ -0,0 +1,31 @@
+const { promisify } = require('util');
+const read = promisify(require('read'));
+const { processConfig } = require('@commercetools-frontend/application-config');
+const CredentialsStorage = require('../utils/credentials-storage');
+const getSessionData = require('../utils/get-session-data');
+
+const credentialsStorage = new CredentialsStorage();
+
+const login = async () => {
+  const applicationConfig = processConfig();
```

Comment:
I was wondering how does the `processConfig` know about the `.env` file in case the user provides one:
```
mc-scripts login --env .env.ctp-gcp-production-eu
```

Reply:
It's implicitly done by the `proxyCommand` on the `cli.js` file

https://github.com/commercetools/merchant-center-application-kit/blob/dd5e33ea58b15e32a2d9aeb1d95cb4a4aec8d069/packages/mc-scripts/src/bin/cli.js#L120-L123